src/frontend/src/pages/CreditsPage/CreditsPage.tsx

```diff
@@ -193,7 +193,8 @@ const CreditsPage: React.FC = () => {
     { name: 'Rup Jaisinghani', color: '#065535' },
     { name: 'Jack Dreifus', color: '#014421' },
     { name: 'Vinay Pillai', color: '#42458e' },
-    { name: 'Will Anderson', color: '#ebb145' }
+    { name: 'Will Anderson', color: '#ebb145' },
+    { name: 'Visisht Kamalapuram', color: '#3083AA'}
```

Comment:
Your prettier test is failing which means that your code is not formatted correctly. To fix it you can add a space right before the closing bracket

Reply:
I fixed the formatting for the code, but I am not sure how to fix the yarn.lock file. Should I just replace new yarn.lock file on my branch with the lines from the old one on master? Sorry and thank you